README.txt

```diff
@@ -21,13 +21,13 @@ or simply:
 
        cabal install
 
-This will install a PortMidi package that contains a Sound.PortMidi module. 
+This will install a PortMidi package that contains a Sound.PortMidi module.
 
 ===================
 Contact Information
 ===================
 
-You may send your bug report and feature request to the package 
```

Comment:
I think this paragraph can be changed to point to github repo: https://github.com/ninegua/PortMidi

Reply:
Done